examples/collection.json

```diff
@@ -0,0 +1,166 @@
+{
+  "type": "Collection",
+  "id": "palsar2-scansar",
+  "stac_version": "1.0.0",
+  "description": "The new PALSAR-2 ScanSAR products have been developed to conform with the CEOS-ARD format specifications for Normalised Radar Backscatter (CARD4L NRB). The NRB products have been subject to ortho-rectification and Radiometric Terrain Correction (RTC), and are provided in the Gamma-0 backscatter convention, which mitigates the variations from diverse observation geometries. The products include HH and HV polarisation backscatter, local incidence angle image and data mask.",
+  "links": [
+    {
+      "rel": "root",
+      "href": "/workspaces/palsar2-scansar/examples/palsar2-scansar.json",
+      "type": "application/json",
+      "title": "ALOS-2 PALSAR-2 ScanSAR"
+    },
+    {
+      "rel": "documentation",
+      "href": "https://www.eorc.jaxa.jp/ALOS/en/dataset/palsar2_l22_e.htm",
+      "type": "application/pdf",
+      "title": "The new PALSAR-2 ScanSAR products have been developed to conform with the CEOS-ARD format specifications for Normalised Radar Backscatter (CARD4L NRB). The NRB products have been subject to ortho-rectification and Radiometric Terrain Correction (RTC), and are provided in the Gamma-0 backscatter convention, which mitigates the variations from diverse observation geometries. The products include HH and HV polarisation backscatter, local incidence angle image and data mask.",
+      "description": "Also includes data usage information"
+    },
+    {
+      "rel": "license",
+      "href": "https://earth.jaxa.jp/policy/en.html",
+      "type": "JAXA Terms of Use of Research Data"
+    },
+    {
+      "rel": "self",
```

Comment:
- in the process of removing

Reply:
@wildintellect I think that we want a create-examples.py for these, similar to https://github.com/stactools-packages/sentinel1/blob/main/scripts/create_examples.py